eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -499,6 +499,18 @@
           "old" : ".*? com\\.azure\\.communication\\.chat\\.models.*",
           "new" : ".*? com\\.azure\\.communication\\.chat\\.models.*",
           "justification": "Migration to azure-json"
+        },
+        {
+          "ignore": true,
+          "code": "java.method.added",
+          "new": "method void com.azure.core.credential.AccessToken::<init>(java.lang.String, java.time.OffsetDateTime, java.time.OffsetDateTime)",
+          "justification": "Added a new constructor to the AccessToken class."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.added",
+          "new": "method java.time.OffsetDateTime com.azure.core.credential.AccessToken::getRefreshAt()",
+          "justification": "Added getRefreshAt method to AccessToken class."
```

Comment:
Adding new methods in a minor version update is not a breaking change. Why is revapi suppression needed here?